readcourse.data

```diff
@@ -0,0 +1,148 @@
+import json
+import os
+from pymongo import MongoClient
+
+class Course:
+	def __init__(self, crn):
+		self.crn = crn
+
+	def __init__(self, **kwargs, *args):
+		self.UID = kwargs.['UID']
+		self.crn = kwargs.['crn']
+		self.course_num = kwargs.['course_num']
+		self.section_num = kwargs.['section_num']
+		self.campus = kwargs.['campus']
+		self.num_credit = kwargs.['num_credit']
+		self.course_title = kwargs.['course_title']
+		self.days = kwargs.['days']
+		self.startTime = kwargs.['startTime']
+		self.endTime = kwargs.['endTime']
+		self.cap = kwargs.['cap']
+		self.wl_cap = kwargs.['wl_cap']
+		self.instructor_name = kwargs.['instructor_name']
+		self.startDate = kwargs.['startDate']
+		self.endDate = kwargs.['endDate']
+		self.location = kwargs.['location']
+		self.attribute = kwargs.['attribute']
+    
+//
+		self.lab = list()
+		self.act = list()
+		self.wl_act = list()
+
+class Instructor:
+	def __init__(self, firstName, middleName, lastName):
+		self.firstName = firstName
+		self.middleName = middleName
+		self.lastName = lastName
+		self.website = ''
+		self.email = ''
+		self.department = list()
```

Comment:
please use `[]` instead `list()`

Reply:
If it change to " self.department = [] ", when using *args or **kwargs, can this become " self.department = args[department]" ?

Reply:
no, just because `[]` are 2 chars, `list()` are 6 chars